package.json

```diff
@@ -11,6 +11,11 @@
     "@babel/core": "^7.4.5",
     "@babel/node": "^7.4.5",
     "@babel/preset-env": "^7.4.5",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^8.0.6",
+    "babel-polyfill": "^6.26.0",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
Does this package is deprecated and replaced by `babel-preset-env` ?

Reply:
Tbh I don't know, it might be useless it was to make unit test works

Reply:
Ok for me ✔️